src/app/utils/payments.ts

```diff
@@ -0,0 +1,16 @@
+import dayjs from "dayjs";
```

Comment:
Is there a way or does it make sense to use this for these as well?:
- https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/Home/DefaultView/index.tsx/#L27-L34
- https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/Home/AllowanceView/index.tsx/#L48-L57

Or maybe in a follow-up PR

Reply:
@rolznz I thought about this, but not enough, because of the different `publisherLink` - now that I think about it more, it could be handled easily as well, and would make sense imo.

I would change it to 

```
export const convertPaymentToTransaction = (
  payment: DbPayment,
  publisherLink?: string
): Transaction => ({
  ...payment,
  id: `${payment.id}`,
  type: "sent",
  date: dayjs(payment.createdAt).fromNow(),
  title: payment.name || payment.description,
  publisherLink: publisherLink || payment.location,
});
```

and apply it with

`const payments = convertPaymentsToTransactions(response.payments, "options.html#/publishers");`

I started refactoring and found this would lead into a further cleanup of the `fetchData` in DefaultView and AllowanceView. For this reason I think it would make sense to refactor in a follow up PR, as you suggest. Should I create a ticket and assign it to me, as I already know what to do and implement it after this PR is merged?

Reply:
>  Should I create a ticket and assign it to me, as I already know what to do and implement it after this PR is merged?

❤️ please do

Reply:
I added https://github.com/getAlby/lightning-browser-extension/issues/2057